src/core/Config.h

```diff
@@ -203,7 +212,7 @@ class Config {
 	
 public:
 	
-	void setActionKey(ControlAction action, int index, InputKeyId key);
+	void setActionKey(const ControlAction action, uint32_t index, InputKeyId newKey);
```

Comment:
There is no reason to add const to a non reference, non-pointer parameter in a function prototype - it has literally no meaning.